src/Header/SetCookie.php

```diff
@@ -378,7 +378,7 @@ public function getMaxAge()
      */
     public function setExpires($expires)
     {
-        if ($expires === null) {
+        if ($expires === null || $expires === '') {
```

Comment:
@Martin-P Expires header and Set-Cookie headers are defined by different rules.

It's wrong assume a constraint for Proxy header affects to Authorization header. The same is applicable for Expires and Set-Cookie